Assets/AppCenter/Plugins/iOS/Crashes/CrashesUnity.mm

```diff
@@ -49,17 +50,25 @@ void appcenter_unity_crashes_disable_mach_exception_handler()
   [MSCrashes disableMachExceptionHandler];
 }
 
-MSErrorReport* app_center_unity_crashes_last_session_crash_report()
+void appcenter_unity_crashes_set_user_confirmation_handler(void(* userConfirmationHandler)())
 {
-    return [MSCrashes lastSessionCrashReport];
+    [MSCrashes setUserConfirmationHandler:^BOOL(NSArray<MSErrorReport *> *_Nonnull errorReports){
+        userConfirmationHandler();
+        return false;
```

Comment:
Why do you not expose return value?
See description for this [here](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/943515eac2dcbce7d2f72b0701f2d99f2df57f22/AppCenterCrashes/AppCenterCrashes/MSCrashes.h#L9)

Reply:
Fixed, thank you